05/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>RWD - 05 Task</title>
+    <link rel="stylesheet" href="css/global.css" />
+    <link rel="stylesheet" media="(min-width: 800px)" href="css/desktop.css" />
+    <link
+      rel="stylesheet"
+      href="node_modules/html5-reset/assets/css/reset.css"
+    />
+  </head>
+  <body>
+    <header class="container">
+      <nav class="section__container nav__section">
+        <div class="nav__logo"></div>
+        <label for="menu-switcher"
+          ><div class="nav__hamburger hamburger--absolute"></div
+        ></label>
+        <input type="checkbox" id="menu-switcher" />
+        <div class="nav__list-container">
+          <ul class="nav__list-elements">
+            <li class="section__ele"></li>
```

Comment:
Tutaj podobnie - skoro jest Element `section__ele` to gdzieś wyżej powinien występować Block `section`